Library/ViewModels/AddNewCardViewModelTests.swift

```diff
@@ -215,13 +215,11 @@ internal final class AddNewCardViewModelTests: TestCase {
   }
 
   func testSetPublishableKey() {
```

Comment:
Maybe we could update the test (or create a new one) that checks if the `stripePublishableKey` is correct for staging and production?

Reply:
This is a bit tricky because the key is different on Circle than on our local machines (because on Circle it skips cloning the `Secrets` repo).